src/patterns/atoms/Text/Text.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import "./Text.scss"
+type Props = {
+    children?: string,
+    size?: string
+}
+
+export const Text: React.FC<Props> = (props: Props) => {
+    const { children, size = 14 } = props
+    return <p className="text" style={{fontSize:`${size}px`}} >{children}</p>
```

Comment:
it's not a good practice to add styles for sizing this way. let's discuss this on our Q&A session

Reply:
implemented 🚀